src/containers/EventContainer/EventInfoTabs.tsx

```diff
@@ -96,6 +99,7 @@ const EventInfoTabs: FC<EventInfoTabsProps> = ({
   }, [eventMinutesItems, votesByEventMinutesItem]);
 
   const infoTabPanes = useMemo(() => {
+    console.log(`calculating tabs for language ${language}`);
```

Comment:
Might want to remove the console logs if you're done with them.

Reply:
Yeah, I wanted to, but it would cause a linting warning. "Remove `language` from the dependency array if it's not being used." But at the same time, I need `language` as a dependency because I wanted the tabs to be re-calculated whenever `language` changes.